sofagym/AbstractEnv.py

```diff
@@ -124,6 +124,8 @@ def __init__(self, config=None):
 
         """
 
+        config = {"dt":0.01}
```

Comment:
What happens to `config` if it is not `None`? It looks like it is replaced in all cases.

Reply:
the ```self.config``` dictionary from the sub-class of the environment gets updated with the ```dt``` argument in ```config```. Otherwise, if ```config``` is ```None``` then ```dt``` is not added at all and it's not defined in ```DEFAULT_CONFIG``` in the sub-class, which is causing the error.